functional-test/src/test/resources/conf/standalone_wildfly.xml

```diff
@@ -317,7 +317,7 @@
     <subsystem xmlns="urn:jboss:domain:jpa:1.1">
       <jpa default-datasource="" default-extended-persistence-inheritance="DEEP"/>
     </subsystem>
-    <subsystem xmlns="urn:jboss:domain:jsf:1.0" default-jsf-impl-slot="mojarra-2.1.29-01"/>
```

Comment:
Correct; it's because the JSF 2.1 module (in
wildfly-module-mojarra-2.1.29-04.zip) now replaces the default slot for JSF.

On 16 October 2015 at 07:42, Carlos Munoz <notifications@github.com> wrote:

> In functional-test/src/test/resources/conf/standalone_wildfly.xml
> <https://github.com/zanata/zanata-server/pull/966#discussion_r42184574>:
>
> > @@ -317,7 +317,7 @@
> >      <subsystem xmlns="urn:jboss:domain:jpa:1.1">
> >        <jpa default-datasource="" default-extended-persistence-inheritance="DEEP"/>
> >      </subsystem>
> > -    <subsystem xmlns="urn:jboss:domain:jsf:1.0" default-jsf-impl-slot="mojarra-2.1.29-01"/>
>
> So, there's no need to identify the specific jsf implementation module
> anymore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/966/files#r42184574>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat